src/app/views/user/user-settings/user-profile/user-profile.component.html

```diff
@@ -174,17 +174,17 @@
 
         <div class="profile-input-table-container">
           <div class="profile-input-table-row-container" *ngFor="let userLanguage of profileForm.value.user_languages">
-            <div class="profile-input-table-row-inner-container" *ngIf="userLanguage.proficiency && userLanguage.proficiency.proficiency !== 5">
+            <div class="profile-input-table-row-inner-container" *ngIf="userLanguage.proficiency">
               <div class="profile-input-table-row-inner-item-container">
                 <div class="profile-input-table-row-inner-item-text">
                   {{userLanguage.language.name}}
                 </div>
-                <div class="profile-input-table-row-inner-icon-container" (click)="onRemoveUserLanguage(userLanguage)">
+                <div class="profile-input-table-row-inner-icon-container" (click)="onRemoveUserLanguage(userLanguage)" *ngIf="userLanguage.proficiency.proficiency !== 5">
```

Comment:
Not here. The ngIf of the grandparent has a check so this html is unreachable with userLanguage.proficiency undefined. Hence it is ok.